packages/synthetix-main/contracts/modules/core/PoolModule.sol

```diff
@@ -67,6 +67,18 @@ contract PoolModule is IPoolModule {
         emit PoolNominationRenounced(poolId, msg.sender);
     }
 
+    function revokePoolNomination(uint128 poolId) external override {
+        Pool.onlyPoolOwner(poolId, msg.sender);
+
+        Pool.Data storage pool = Pool.load(poolId);
+
+        address nominatedOwner = pool.nominatedOwner;
+
+        Pool.load(poolId).nominatedOwner = address(0);
+
+        emit PoolNominationRevoked(poolId, nominatedOwner);
+    }
```

Comment:
`revokePoolNomination` is redundant as there is a similar function already:
`function renouncePoolOwnership(uint128 poolId) external override {
        Pool.onlyPoolOwner(poolId, msg.sender);

        Pool.load(poolId).nominatedOwner = address(0);

        emit PoolOwnershipRenounced(poolId, msg.sender);
    }`

Reply:
hmm you are right. Maybe we should get rid of the event?

Reply:
@noahlitvin do we want pool nominated owners be able to renounce their nomination?

Reply:
hold on, I'm I stupid or does this func `renouncePoolOwnership` do the same than `renouncePoolNomination `